mappings/net/minecraft/world/gen/DensityFunction.mapping

```diff
@@ -0,0 +1,42 @@
+CLASS net/minecraft/unmapped/C_kzwapyud net/minecraft/world/gen/DensityFunction
+	METHOD m_apksyljd squeeze ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_gmrgdcrk quarterNegative ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_hfnqnnds minValue ()D
+	METHOD m_idbpaztj halfNegative ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_jscnkdwi cube ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_ngodkcgz clamp (DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 1 minValue
+		ARG 3 maxValue
+	METHOD m_onnxbyiv fillArray ([DLnet/minecraft/unmapped/C_kzwapyud$C_rznisoqn;)V
+		ARG 1 array
+		ARG 2 context
+	METHOD m_povvwofi maxValue ()D
+	METHOD m_ssiakess compute (Lnet/minecraft/unmapped/C_kzwapyud$C_rbrbrmhy;)D
+		ARG 1 functionContext
+	METHOD m_tomueclj mapAll (Lnet/minecraft/unmapped/C_kzwapyud$C_zdlfgpyk;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 1 visitor
+	METHOD m_tttpbozk square ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_ugbzkmol abs ()Lnet/minecraft/unmapped/C_kzwapyud;
+	CLASS C_fntlxgzz SimpleFunction
+	CLASS C_pjhchjte SinglePointContext
```

Comment:
Is this a record? If so, the parameter names don't match the component names, and the methods aren't named

Reply:
It was, my bad. Not sure why the methods weren't named 🤔